import.py

```diff
@@ -316,6 +360,20 @@ def modelClassGen(stem, sf, keyField, srs, shapeType, shapefileGroup):
 
 
 
+def modelClassGenRaster(stem, rst, bandNumber, shapefileGroup):
+  text  = "class " + stem + "(models.Model):\n"
+  text += "    def getGroup():\n"
+  text += "        return ShapefileGroup.objects.get_or_create(name='" + shapefileGroup + "')[0]\n\n"
```

Comment:
ha, I guess I am going to have to rename `ShapefileGroup` eventually!

Reply:
Yeah, there's a few ways that we've slid into now using "shapefile" to mean "dataset" or "data source".